src/cryptoadvance/specter/templates/includes/language/language_select.jinja

```diff
@@ -0,0 +1,10 @@
+{#
+	Relies on `supported_languages=app.supported_languages` being injected into the context
+#}
+<select name="language_code" onchange="changeLanguage(this)">
+	{% for language_code, language in supported_languages.items() %}
+		<option value="{{ language_code }}" {% if session["language_code"] == language_code %}selected{% endif %}>{{ language }}</option>
+	{% endfor %}
+	<option value="de" disabled {% if session["language_code"] == "de" %}selected{% endif %}>Deutsch</option>
```

Comment:
I guess these 2 should be removed since the languages are fetched autoomatically

Reply:
Done!